### ISSUE TITLE
***BUG**** - Breaking Change shows as Failed but Signoff API Spec Readiness shows as not required 

### ISSUE THREAD

**Issue by ladonnaq** [Azure/azure-sdk-tools#6327]
New service working on the initial release of their beta SDKs for management plane.  They have not published their initial preview API version.  They are working on finalizing the changes to preview API version in the private repo and will publish at public preview launch. 

The PR met API readiness for signoff, which looks to be correct as there is no ARM Review required for these changes and the breaking changes would not be relevant for this PR.  However, it is confusing b/c the breaking change error and ARM review not available are seen earlier in the flow.  The service team engineer pinged me b/c he was confused by these results. 

- [ ] **Suggested requirement:** If ARM Review or Breaking Change are not relevant for the PR, then show not required on the earlier screens.  

![image](https://github.com/Azure/azure-sdk-tools/assets/67656762/e6460b69-57a8-4a1f-87fd-5e2ab0e12b15)
![image](https://github.com/Azure/azure-sdk-tools/assets/67656762/f8aeee82-8453-4a65-adec-27db39328388)

**Comment by maririos** [comment]
@praveenkuttappan how does the Sign Off page knows what are the things that are required? is there a way to take that logic into account when showing the other views?

**Comment by praveenkuttappan** [comment]
@ladonnaq @maririos Validation statuses and breaking change detection statuses are disaplyed as per the check run status for the given PR. If a Spec PR has breaking change detection failed then release planner will show it as failed and there is no automatic logic to determine whether a failure is valid or applicable. My understanding is that these errors need to be suppressed on GitHub PR using suppression label by the approvers. Once suppressed on PR, statuses on release planner will automatically get cleared.

**Comment by maririos** [comment]
So in this case, there was a fialure, but somehow the SignOff page knows it is not required. How?

**Comment by ladonnaq** [comment]
> @ladonnaq @maririos Validation statuses and breaking change detection statuses are disaplyed as per the check run status for the given PR. If a Spec PR has breaking change detection failed then release planner will show it as failed and there is no automatic logic to determine whether a failure is valid or applicable. My understanding is that these errors need to be suppressed on GitHub PR using suppression label by the approvers. Once suppressed on PR, statuses on release planner will automatically get cleared.

True this PR seems to be related to commit(s) that did not require ARM review or breaking changes as none of the labels are added.  In this scenario, the release planner app determined on the sign-off screen/step that neither of these reviews were required.  How was it determined that "not required" was the appropriate status for the signoff screen?  I believe this is correct for this particular PR.  If the signoff screen logic can determine if a validation is not required, then we should not show an error earlier in the UI flow.  Instead it should just skip the validation for that section if it is not truly required.

**Comment by praveenkuttappan** [comment]
@ladonnaq I will talk to Konrad about this. There is a breaking change failure as per the PR https://github.com/Azure/azure-rest-api-specs-pr/pull/12081  checks and this failure was not suppressed on the PR. But PR does not have any label to indicate about this breaking change status or approval and summary screen still uses the label. I can fix the summary screen to match breaking change tab status and I would like to know why was this PR allowed to merge even when check was failed. So if this is allowed then it's probably better to mark that check as suppressed automatically.

**Comment by maririos** [comment]
@praveenkuttappan what you are saying is that the app is correctly reflecting what happened in the PR and as a process, there was something off in the way the PR has handled and merged which is why you need to check with Konrad?

**Comment by praveenkuttappan** [comment]
@maririos Exactly. PR has a failed breaking change check and app reflects that in breaking change section. Summary page shows breaking change as no required if PR is mergeable. But my question is, ideally a failed check should have been neutralized before that PR is allowed to merge so that check is marked as completed and it will reflect in our app correctly.

Currently developer can get same confusion if you look at the PR in question, My PR failed breaking change detection but it was allowed to merge. So why is that check even in failed status?

I prefer not to make further logic to handle this in release planner side until there is a clarity on this case if this is some edge case about Model validation change.

**Comment by maririos** [comment]
> I prefer not to make further logic to handle this in release planner side until there is a clarity on this case if this is some edge case about Model validation change.

Agree. I am going to add the Specs label to redirect this question to them

**Comment by ladonnaq** [comment]
@maririos Can this issue be closed?

**Comment by maririos** [comment]
> @maririos Mariana Rios Flores FTE Can this issue be closed?

This is outside of our scope. It is now in Konrad's so I will leave it up to him. But looking at the history it might still need to happen. 
@ladonnaq I removed it from your project so you don't have to track it
